docs/docs/developers/guides/js_apps/deploy_contract.md

```diff
@@ -23,6 +23,7 @@ Compile the contract:
 
 ```bash
 aztec-nargo compile
+aztec-compile-contract
```

Comment:
both of these steps are needed now? the naming is confusing. based on the name here i would expect these steps to be doing the same thing